mongodb/intro/crud-query-collections/query-collection-3.md

```diff
@@ -0,0 +1,172 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+
+aspects:
+  - new
+
+type: normal
+
+category: must-know
+
+standards:
+  javascript.crud.3: 10
+  javascript.comparison-operators.0: 10
+  javascript.comparison-operators.1: 10
+  javascript.comparison-operators.2: 10
+  javascript.comparison-operators.3: 10
+  javascript.logical-operators.0: 10
+
+links:
+
+---
+# Querying a collection using $and
+---
+## Content
+
+In the previous insight, we discussed how to query using the `$lt`, `$gt`, `$lte` and `$gte` operators.
+
+Now, we are going to discuss how to find a range between two values using a combination of the 4 previously explained operators and the `$and` logical operator.
+
+The `$and` logical operator is used to join 2 or more expressions together to query data more precisely. When we query with the `$and` operator, no matter the number of expressions located inside, all expressions have to evaluate to true.
+
+Let's say we want to find all the pokémon whos `power` ranges between 250 and 350, we would need to use 2 or more of the above operators together with the `$and` operator to achieve so. If we would use them without `$and`, it would not give a range between the two values, but rather all values lower than and greater than the specified ones.
```

Comment:
```suggestion
Let's say we want to find all the pokémons whose `power` ranges between 250 and 350, we would need to use 2 or more of the above operators together with the `$and` operator to achieve so. If we would use them without `$and`, it would not give a range between the two values, but rather all values lower than and greater than the specified ones.
```